dotcom-rendering/src/web/components/SignInGate/SignInGateSelector.tsx

```diff
@@ -152,27 +170,51 @@ export const SignInGateSelector = ({
 		if (gateVariant && currentTest) {
 			// eslint-disable-next-line @typescript-eslint/no-floating-promises
 			gateVariant
-				?.canShow(CAPI, !!isSignedIn, currentTest)
+				?.canShow({
+					isSignedIn: !!isSignedIn,
```

Comment:
Probably out of scope for this PR, but seeing this made me wonder - do we prefer `!!` vs `Boolean()`? I kind of like how explicit and obvious `Boolean()` is. Maybe something for the JS/TS recommendations.

Reply:
Oh interesting. I personally didn't know what `Boolean()` was, or at least not enough to be confident using it. Clearly though if you put them side by side `Boolean` makes more sense that two exclamation marks!!

I doubt there's one good option here. The thing that makes most sense to you is based on your developer history which is different for everyone. I agree though that this should be centralised and a dept. level decision made so that we can move on from stuff like this. I love how Prettier removed these issues for formatting, so maybe we can setup something similar for this?